server/bin/pbench-audit-server.sh

```diff
@@ -772,13 +772,17 @@ fi
 
 log_finish
 
-# send it
-subj="$PROG.$TS($PBENCH_ENV)"
-cat << EOF > ${index_content}
+subj="$PROG.$TS ($PBENCH_ENV)"
+if [[ -s ${report} ]]; then
+    cat << EOF > ${index_content}
 $subj
-EOF
 
-cat ${report} >> ${index_content}
+EOF
+    cat ${report} >> ${index_content}
+else
+    printf "${subj} - Successful audit, nothing to report.\n" > ${index_content}
+fi
```

Comment:
So ... you broke the gold tests here, and didn't update them.

![image](https://user-images.githubusercontent.com/61761529/121681890-588e8e80-ca89-11eb-8e6c-0cbb8cf5a68a.png)

Reply:
Bummer, thanks for pointing that out!

Reply:
You better not fix this!  I just reviewed all those gold files!!!